components/widgets/menu/menu_wrapper_animation.jsx

```diff
@@ -6,7 +6,7 @@ import React from 'react';
 import {tween, styler} from 'popmotion';
 import {CSSTransition} from 'react-transition-group';
 
-const ANIMATION_DURATION = 180;
+const ANIMATION_DURATION = 80;
```

Comment:
This change affect both animations, you are changing the speed in the rest of the menus, not only in mobile

Reply:
Yup. I know.